content/en/platform/corda/1.6/cenm/zone-service.md

```diff
@@ -71,6 +76,109 @@ The full list of configuration options follows below:
 - `--auth-leeway`: Defines the amount of time, in seconds, allowed when checking JSON Web Token (JWT) issuance and expiration times. Required unless authentication and authorisation are disabled. R3 recommends a default time of **10 seconds**.
 - `--working-dir`: Defines the working directory to the specified directory. The service will look for files in that directory. This means certificates, configuration files etc. should be under the working directory. If not specified it will default to the current working directory (the directory from which the service has been started).
 
+## Configuration
+
+The main elements that you must configure for the Zone Service are:
+
+* [Database](#database)
+* [Admin RPC Interface](#admin-rpc-interface)
+
+### Database
+
+The Zone Service is backed by a SQL database which is used to store information such as the configuration files for each service for the Angel Service to download.
+
+You must include the connection settings within the `database` configuration block in the configuration file. The main options that should be included here are:
+
+* `driverClassName`: The database driver class name (for example, `com.microsoft.sqlserver.jdbc.SQLServerDriver` for Microsoft SQL Server, `org.postgresql.Driver` for PostgreSQL).
+* `jdbcDriver`: The path to the appropriate JDBC driver JAR (for example, `path/to/mssql-jdbc-7.2.2.jre8.jar`).
+* `url`: The connection string for the database.
+* `user`: The user name for the database.
+* `password`: The password for the database.
+
+#### Database Setup
+
+The database can either be set up prior to running the Zone Service or it can be automatically prepared on start-up via the built-in migrations. To enable the running of database migrations on start-up, you must set the optional `runMigration` parameter within the `database` configuration to `true`.
```

Comment:
```suggestion
The database can either be set up prior to running the Zone Service, or it can be automatically prepared on start-up via the built-in migrations. To enable the running of database migrations on start-up, you must set the optional `runMigration` parameter within the `database` configuration to `true`.
```